core/templates/dev/head/pages/email_dashboard/EmailDashboardDataService.js

```diff
@@ -11,11 +11,10 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
```

Comment:
Please preserve the newline that used to be below this one. In general, it's a good idea to check the "Files Changed" tab on GitHub to make sure that the diff is exactly what you want it to be :)